### PR TITLE
Added linear search.

### PR DESCRIPTION
**Pull request by sgrG24**
Language used c++.

This Pull Request resolves Issue #425 

Description:
In this, I added linear_search implementation in c++ language.
please review this and if any error or any correction is there, i will resolve it.

### CODE REVIEW

File: Linear_search/Linear_search.cpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    for (i=0; i<n; i++)
+        if (arr[i] == x)
+         return i;
+    return -1;
+}
+
+//Driver program
+int main()
+{
+	int a[]={22,34,11,56,432,44};                  //Declaring array
```
Comment (salman-bhai) [inline Linear_search/Linear_search.cpp]: Don't hard-code the array! Take input from the user!

File: Linear_search/Linear_search.cpp [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	int size = sizeof(a)/sizeof(a[0]);             // finding numbers of elements in array 
+	//calling search function
+	int p = search(a, size,11);
+	if(p!=-1)
+	cout<<"Element found";
```
Comment (salman-bhai) [inline Linear_search/Linear_search.cpp]: Please have some indentation here!

File: Linear_search/Linear_search.cpp [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+	else
+	cout<<"Element not found";
```
Comment (salman-bhai) [inline Linear_search/Linear_search.cpp]: Please have some indentation here!

File: Linear_search/Linear_search.cpp (hunk 1 above)
Comment (sgrG24) [reply Linear_search/Linear_search.cpp]: Ok i will update this.

File: Linear_search/Linear_search.cpp (hunk 2 above)
Comment (sgrG24) [reply Linear_search/Linear_search.cpp]: Okay.

**Comment by sgrG24** [comment]
I will solve all this issue and update my code. Thanks for your reviews.

**Comment by salman-bhai** [comment]
Hi, always better to comment after you resolve the changes!

**Comment by sgrG24** [comment]
Ok, I will take care of this. I update all the changes, still if there is any issue or correction, i will solve it.

**Pull request merged**